src/common/disk.rs

```diff
@@ -133,7 +134,9 @@ impl Disk {
         self.inner.is_read_only()
     }
 
-    /// Updates the disk' information.
+    /// Updates the disk' information with everything loaded.
+    ///
+    /// Equivalent to [`Disk::refresh_specifics(DiskRefreshKind::everything())`].
```

Comment:
I don't think this link is generated as expected.

Reply:
fixed, thank you